src/usr/local/www/firewall_rules.php

```diff
@@ -328,6 +328,12 @@ function delete_nat_association($id) {
 	$showblockbogons = true;
 }
 
+if (isset($config['system']['webgui']['roworderdragging'])) {
+	$rules_header_text = 'Rules';
+} else {
+	$rules_header_text = 'Rules (Drag to Change Order)';
+}
```

Comment:
Please add `gettext()` calls on strings and remove from variable

Reply:
Ok, you got it